quiz/mobile/__tests__/App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import renderer from "react-test-renderer";
+
+import App from "../App";
+
+describe("<App />", () => {
+    it("has 1 child", () => {
+        const tree = renderer.create(<App />).toJSON();
+        expect(tree.children.length).toBe(1);
+    });
+});
+
+it("renders correctly", () => {
+    const tree = renderer.create(<App />).toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
Snapshot testing in most cases is pretty usless, beacause it hardly ever finds bugs or unwanted behaviour.